service-mesh/enable-service-mesh.sh

```diff
@@ -12,7 +12,33 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 kubectx central
-bash <( gsutil cat gs://anthos-workshop/csm-alpha-onboard.sh )
-kubectl delete po $(kubectl get pod -n istio-system -l app=telemetry -o json | jq -r '.items[].metadata.name') -n istio-system
 
+# Enable the required GCP APIs for the topology view
+gcloud services enable iamcredentials.googleapis.com contextgraph.googleapis.com --project ${PROJECT}
 
+# Add a csm label for your cluster
+gcloud container clusters update ${CLUSTER_NAME} --zone ${CLUSTER_ZONE} --update-labels csm=
```

Comment:
This is/should be on creation not as an update

Reply:
Agree, the initial idea would be to create the cluster without it and add only in the ASM part. But since everything is bootstrapped at the same time, it makes sense.